### PR TITLE
Update assignment 2 tests

### PR DESCRIPTION
**Pull request by ironore15**
Add test cases for the boundary values before overflow.

### CODE REVIEW

File: src/assignments/assignment02_grade.rs [hunk 1]
```diff
@@ -73,6 +74,8 @@ mod test {
         assert_eq!(fibonacci(6), 13);
         assert_eq!(fibonacci(7), 21);
         assert_eq!(fibonacci(50), 20365011074);
+        assert_eq!(fibonacci(92), 12200160415121876738);
+
```
Comment (minseongg) [inline src/assignments/assignment02_grade.rs:78]: Please remove unnecessary newline

Comment (ironore15) [reply src/assignments/assignment02_grade.rs]: Done. Thanks for reviewing.

File: src/assignments/assignment02_grade.rs [hunk 2]
```diff
@@ -52,6 +52,7 @@ mod test {
...
         assert_eq!(chooses(6), vec![1, 6, 15, 20, 15, 6, 1]);
+        assert_eq!(chooses(67), vec![1, 67, 2211, 47905, 766480, 9657648, 99795696, 869648208, 6522361560, 42757703560, 247994680648, 1285063345176, 5996962277488, 25371763481680, 97862516286480, 345780890878896, 1123787895356412, 3371363686069236, 9364899127970100, 24151581961607100, 57963796707857040, 129728497393775280, 271250494550621040, 530707489338171600, 972963730453314600, 1673497616379701112, 2703342303382594104, 4105075349580976232, 5864393356544251760, 7886597962249166160, 9989690752182277136, 11923179284862717872, 13413576695470557606, 14226520737620288370, 14226520737620288370, 13413576695470557606, 11923179284862717872, 9989690752182277136, 7886597962249166160, 5864393356544251760, 4105075349580976232, 2703342303382594104, 1673497616379701112, 972963730453314600, 530707489338171600, 271250494550621040, 129728497393775280, 57963796707857040, 24151581961607100, 9364899127970100, 3371363686069236, 1123787895356412, 345780890878896, 97862516286480, 25371763481680, 5996962277488, 1285063345176, 247994680648, 42757703560, 6522361560, 869648208, 99795696, 9657648, 766480, 47905, 2211, 67, 1]);
```
Comment (minseongg) [inline src/assignments/assignment02_grade.rs:55]: Could you please apply `cargo fmt --all`?

Comment (ironore15) [reply src/assignments/assignment02_grade.rs]: Done.

**Comment by minseongg** [comment]
Thanks!
